documentation/frontend/development.md

```diff
@@ -0,0 +1,215 @@
+# Development
+
+This [Next.js](https://nextjs.org) application can be run natively (or locally)
+
+**Running in locally is the default**, but it can be useful to run the built images in order to troubleshoot and to connect directly with the local API application for development.
+
+## Local (non-Docker)
+
+### 🏗️ Development version
+
+Run `npm install && npm run dev` to install and start the application.
+
+### 🏛️ "Built" version
+
+The Next.js frontend application is exported for production using [next build](https://nextjs.org/docs/app/api-reference/cli/next#next-build-options). To recreate this locally, outside of the container, run the following:
+
+- `npm run build` - Builds the production Next.js bundle
+- `npm start` - Runs the Next.js server, after building the production bundle
```

Comment:
is it worth noting somewhere that our deployed application runs in standalone mode with `node server.js` (or `node .next/standalone/server.js`) rather than `next start`?